qa/tasks/rgw.py

```diff
@@ -143,6 +143,35 @@ def start_rgw(ctx, config, clients):
                 '--rgw_crypt_vault_token_file', token_path
             ])
 
+            clientcert = ctx.ssl_certificates.get('kmip-client')
```

Comment:
```
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_fd5c31669e51177d3c57bc0420279c5d1e944497/teuthology/contextutil.py", line 31, in nested
    vars.append(enter())
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/teuthworker/src/github.com_ceph_ceph-c_63a8334b02d7dac9af12d1cf5f3ef0666b46826e/qa/tasks/rgw.py", line 146, in start_rgw
    clientcert = ctx.ssl_certificates.get('kmip-client')
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_fd5c31669e51177d3c57bc0420279c5d1e944497/teuthology/config.py", line 259, in __getattr__
    raise AttributeError(name)
AttributeError: ssl_certificates
```
it looks like octopus doesn't have the `kmip` feature at all, sorry - closing this and the associated tracker